lib/tasks/populate.rake

```diff
@@ -18,6 +18,7 @@ namespace :populate do
       # Some parts of the app expect members to have an application with a valid processed_at date.
       if %w[member key_member voting_member].include? user.state
         user.application.processed_at = (1 + rand(5)).days.ago
+        user.save
```

Comment:
This should use `.save!` so that it will throw an exception on fail instead of just returning false.

Reply:
please rereview